patches/minecraft/net/minecraft/tileentity/LockableTileEntity.java.patch

```diff
@@ -10,16 +10,16 @@
 +      return new net.minecraftforge.items.wrapper.InvWrapper(this);
 +   }
 +
-+   @javax.annotation.Nullable
++   @javax.annotation.Nonnull
```

Comment:
This annotation can just be removed. It was Nullable to override the vanilla's package level ones. Back before the cap optional rewrite.